<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemoteBLASTDialog</class>
 <widget class="QDialog" name="RemoteBLASTDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>523</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Search Through a Remote Database</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinAndMaxSize</enum>
   </property>
   <item>
    <widget class="QTabWidget" name="optionsTab">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGeneralOptions">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="title">
       <string>General options</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinAndMaxSize</enum>
       </property>
       <item>
        <layout class="QGridLayout" name="layoutSearchOptions">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinAndMaxSize</enum>
         </property>
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="horizontalSpacing">
          <number>15</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="lblDatabase">
           <property name="text">
            <string>Select the search type:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="dataBase">
           <item>
            <property name="text">
             <string>blastn</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>blastp</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>cdd</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QCheckBox" name="shortSequenceCheckBox">
           <property name="text">
            <string>Search for short, nearly exact matches</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lblEvalue">
           <property name="text">
            <string>Expectation value:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QCheckBox" name="megablastCheckBox">
           <property name="text">
            <string>Megablast</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="lblQuantity">
           <property name="toolTip">
            <string>The maximum number of results</string>
           </property>
           <property name="text">
            <string>Results limit:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSpinBox" name="quantitySpinBox">
           <property name="toolTip">
            <string>The maximum number of results</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="maximum">
            <number>5000</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
           <property name="value">
            <number>20</number>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="evalueSpinBox">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="decimals">
            <number>6</number>
           </property>
           <property name="minimum">
            <double>0.000001000000000</double>
           </property>
           <property name="maximum">
            <double>100000.000000000000000</double>
           </property>
           <property name="value">
            <double>10.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="layoutDb">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetMinAndMaxSize</enum>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="lblDb">
           <property name="text">
            <string>The database:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="dbComboBox">
           <property name="currentIndex">
            <number>2</number>
           </property>
           <item>
            <property name="text">
             <string>Human genomic plus transcript - est_human</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Mouse genomic plus transcript - est_mouse</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Nucleotide collection - nr</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Reference nRNA sequences - refseq_rna</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Reference genomic seqence - refseq_genomic</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>NCBI genomes - chromosome</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Express sequence tags - est</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Non-human, non-mouse ESTs - est_other</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Genomic survey sequences - gss</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>High throughput genomic sequences - HTGS</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Patent sequences - pat</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Protein data bank - pdb</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Human ALU repeat elements - alu_repeats</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Sequence tagged sites - dbsts</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Whole-genome shotgun reads - wgs</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Environmental samples - env_nt</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="layoutDbDescription">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetMinAndMaxSize</enum>
         </property>
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="lblDbDescription">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>The database description:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="teDbDescription">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="html">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Lucida Grande'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="layoutAnnotations">
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layoutTimeout">
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <item>
          <spacer name="spacerTimeout">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="lblTimeout">
           <property name="text">
            <string>Search timeout</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="retrySpinBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="suffix">
            <string> min</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>6000</number>
           </property>
           <property name="singleStep">
            <number>1</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabAdvancedOptions">
      <attribute name="title">
       <string>Advanced options</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinAndMaxSize</enum>
       </property>
       <item>
        <layout class="QGridLayout" name="layoutAdvancedOptions">
         <item row="0" column="1">
          <widget class="QComboBox" name="wordSizeComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>2</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>7</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>11</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>15</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="lblWordSize">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Word size</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="lblCosts">
           <property name="text">
            <string>Gap costs</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QComboBox" name="costsComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>2</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>11 1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>5 2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4 2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1 2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>0 2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3 1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2 1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1 1</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lblScores">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Match scores</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="scoresComboBox">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>1 -2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1 -3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1 -4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2 -3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4 -5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1 -1</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="lblMatrix">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Matrix</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="matrixComboBox"/>
         </item>
         <item row="5" column="1" colspan="4">
          <widget class="QLineEdit" name="phiPatternEdit"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="lblPhiPattern">
           <property name="text">
            <string>PHI pattern</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <spacer name="spacerAdvancedOptions1">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lblEntrezQuery">
           <property name="text">
            <string>Entrez query</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="4">
          <widget class="QLineEdit" name="entrezQueryEdit"/>
         </item>
         <item row="3" column="3">
          <widget class="QLabel" name="lblService">
           <property name="text">
            <string>Service</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <spacer name="spacerAdvancedOptions2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="4">
          <widget class="QComboBox" name="serviceComboBox">
           <item>
            <property name="text">
             <string>plain</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>psi</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>phi</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layoutFiltersAndMasks">
         <item>
          <widget class="QGroupBox" name="gbFilters">
           <property name="title">
            <string>Filters</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QCheckBox" name="lowComplexityFilterCheckBox">
              <property name="text">
               <string>Low complexity filter</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="repeatsCheckBox">
              <property name="text">
               <string>Human repeats filter</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="gbMasks">
           <property name="title">
            <string>Masks</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QCheckBox" name="lookupMaskCheckBox">
              <property name="text">
               <string>Mask for lookup table only</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="lowerCaseCheckBox">
              <property name="text">
               <string>Mask lower case letters</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="gbFilterResults">
         <property name="title">
          <string>Filter results</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QGridLayout" name="layoutFilterResults">
            <item row="0" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Filter by</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QCheckBox" name="defCheckBox">
              <property name="text">
               <string>def (filter by definition of annotations)</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QRadioButton" name="scoreRadioButton">
              <property name="text">
               <string>Score</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QCheckBox" name="accessionCheckBox">
              <property name="text">
               <string>accession</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QCheckBox" name="idCheckBox">
              <property name="text">
               <string>id</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QRadioButton" name="evalueRadioButton">
              <property name="text">
               <string>EValue</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Select result by</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>1</width>
           <height>1</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="9"/>
 <tabstops>
  <tabstop>optionsTab</tabstop>
  <tabstop>dataBase</tabstop>
  <tabstop>shortSequenceCheckBox</tabstop>
  <tabstop>megablastCheckBox</tabstop>
  <tabstop>quantitySpinBox</tabstop>
  <tabstop>evalueSpinBox</tabstop>
  <tabstop>dbComboBox</tabstop>
  <tabstop>teDbDescription</tabstop>
  <tabstop>retrySpinBox</tabstop>
  <tabstop>wordSizeComboBox</tabstop>
  <tabstop>scoresComboBox</tabstop>
  <tabstop>costsComboBox</tabstop>
  <tabstop>entrezQueryEdit</tabstop>
  <tabstop>matrixComboBox</tabstop>
  <tabstop>serviceComboBox</tabstop>
  <tabstop>phiPatternEdit</tabstop>
  <tabstop>lowComplexityFilterCheckBox</tabstop>
  <tabstop>repeatsCheckBox</tabstop>
  <tabstop>lookupMaskCheckBox</tabstop>
  <tabstop>lowerCaseCheckBox</tabstop>
  <tabstop>accessionCheckBox</tabstop>
  <tabstop>defCheckBox</tabstop>
  <tabstop>idCheckBox</tabstop>
  <tabstop>evalueRadioButton</tabstop>
  <tabstop>scoreRadioButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
